antora.yml

```diff
@@ -1,5 +1,5 @@
 name: design-center
 title: Design Center
-version: master
```

Comment:
@rddheath The value of this must be "master". It's a bit of a quirk in Antora, but it's not actually the name of the branch. "master" here just means "only version"

Reply:
The value is currently "master".